grabcut/grabcut.py

```diff
@@ -320,8 +325,10 @@ def view(self):
         dmap = hv.DynamicMap(self.extract_foreground)
         dmap = hv.util.Dynamic(dmap, operation=self._filter_contours)
         dmap = hv.util.Dynamic(dmap, operation=self._simplify_contours)
-        return (self.image.opts(**options) * self.bg_paths * self.fg_paths +
-                dmap.opts(**options))
+        return pn.Row(
```

Comment:
If two freehanddraw tools collide, https://github.com/holoviz/holoviews/issues/6140

`merge_tools` also broken :'( https://github.com/holoviz/holoviews/issues/6139

Reply:
Simon mentioned using different tooltip names and it worked.
https://github.com/holoviz/holoviews/issues/6140